### DISCUSSION TITLE
Changes to xdrip+ watchface Large

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#3091] [Ideas]
I know real estate is limited on the watchface, however would it be possible to put the LOWEST BG reading and the HIGHEST BG reading (during the period shown on the watch graph maybe) on the status bar? this would be invaluable information to have. you could make the current bg reading (font) at the top of the watchface a bit smaller to gain some more real estate.
ALSO it would be nice if the minutes since last reading  (and lowest and highest number could be displayed in BOLD font to make it more readable....

**Comment by Navid200** [comment]
I don't think we have any active developer looking after WearOS.
I have no experience with it.  Someone who does can open a PR.

**Comment by mprosenberg** [comment]
What watch do you have?  I have designed a couple of watch faces that have the information on the watch. It should work on Wear Os or Tizen based watches. I have the classic Galaxy watch, and posted two of the designs on WatchMaker.  The face of the watch is shown here, and then if you tap the bottom of the screen you get the second screen, and can select the 2 hr, 4 hr, or 6 hr information.
![Screenshot_20231210_110812_WatchMaker](https://github.com/NightscoutFoundation/xDrip/assets/58182423/da499069-f9b9-4118-a487-2c5e86e5c4bf)     ![Screenshot_20231210_110820_WatchMaker](https://github.com/NightscoutFoundation/xDrip/assets/58182423/1ee6cd13-4bca-4b50-acaa-2aa9a0759d53)    

I have other watch faces I can send.

Mike

**Comment by Npavkov** [comment]
I have galaxywatch4 and galaxywatch5.....how do I get the 2nd watchface (BIG NUMBERS) if I do not have watchmaker, on the watches?
